pkg/frontend/util.go

```diff
@@ -917,6 +917,19 @@ func convertRowsIntoBatch(pool *mpool.MPool, cols []Column, rows [][]any) (*batc
 			if err != nil {
 				return nil, nil, err
 			}
+		case types.T_enum:
```

Comment:
look at the code, we new vData, copy into vData, then copy into vector.   Why.

extend vector and just copy into the vector directly?

Reply:
Other data types do the same thing

Reply:
https://github.com/matrixorigin/matrixone/issues/17155

Please create this issue yourself next time :).

Reply:
ok,thanks